src/lava/proc/lif/process.py

```diff
@@ -5,29 +5,50 @@
 import numpy as np
 import typing as ty
 
+from lava.magma.core.learning.learning_rule import (
+    LoihiLearningRule,
+    Loihi2FLearningRule,
+)
+from lava.magma.core.process.process import AbstractProcess, LogConfig
 from lava.magma.core.process.process import LogConfig, AbstractProcess
 from lava.magma.core.process.variable import Var
 from lava.magma.core.process.ports.ports import InPort, OutPort
 
+from lava.magma.core.process.neuron import LearningNeuronProcess
+
+from lava.proc.learning_rules.r_stdp_learning_rule import RewardModulatedSTDP
```

Comment:
I dont think we need this import anymore. The RSTDPLIF doesn't seem to be defined in this file.

Reply:
done